views/admin/orm/views/actions/index.php

```diff
@@ -27,10 +27,12 @@
 			</td>
 		</tr>
 <?php echo '<?php endforeach; ?>'; ?>
-<?php echo $pagination;?>
+
 	</tbody>
 </table>
 
+<?php echo '<?php'; ?> echo $pagination <?php echo '?>'; ?>
```

Comment:
I moved the code to render pagination here, outside of `table` block. And this is the correct code generate the code to render pagination